src/Runner.Listener/Checks/ActionsCheck.cs

```diff
@@ -47,6 +48,7 @@ public async Task<bool> RunCheck(string url, string pat)
                 githubApiUrl = urlBuilder.Uri.AbsoluteUri;
                 actionsTokenServiceUrl = "https://vstoken.actions.githubusercontent.com/_apis/health";
                 actionsPipelinesServiceUrl = "https://pipelines.actions.githubusercontent.com/_apis/health";
+                actionsPipelinesServicePostUrl = "https://pipelines.actions.githubusercontent.com/_apis/distributedtask/pools/1/agents";
```

Comment:
is this an unauthenticated response? and we're just checking response headers?

Reply:
we just check the response header to make sure it was from the Actions service, the request will 401 or 404.